src/pages/filter/components/input-radio/input-radio.component.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { InpSwitch } from '../../../comparison/interfaces/filter.interface';
```

Comment:
I would recommend to rename `InptSwitch` to the `InputRadioProps`. The interface name should be self-described. And match the component name (not always)